### PR TITLE
Added note about characters not loading

### PR DESCRIPTION
**Pull request by keithhc2**
Signed-off-by: keithhc2 <keithhc2@users.noreply.github.com>

#257 
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by davidcui1225 — APPROVED** [review]

File: _dashboards/reporting.md [hunk 1]
```diff
@@ -55,3 +55,7 @@ This problem can occur for two reasons:
 - You don't have the correct version of `headless-chrome` to match the operating system on which OpenSearch Dashboards is running. Download the correct version [here](https://github.com/opensearch-project/dashboards-reports/releases/tag/chromium-1.12.0.0).
 
 - You're missing additional dependencies. Install the required dependencies for your operating system from the [additional libraries](https://github.com/opensearch-project/dashboards-reports/blob/main/dashboards-reports/rendering-engine/headless-chrome/README.md#additional-libaries) section.
+
+### Characters not loading in reports
+
+If your report contains any UTF-8 encoded characters that are more than three bytes, you may encounter an issue where the characters look fine in your browser, but they do not load in your generated reports. Install the [missing font dependencies](https://github.com/opensearch-project/dashboards-reports#missing-font-dependencies), and then generate your reports again.
```
Comment (zhongnansu) [inline _dashboards/reporting.md:61]: I think it's more than 2 bytes, right? @davidcui1225 
https://design215.com/toolbox/utf8-3byte-characters.php

Comment (ashwinkumar12345) [reply _dashboards/reporting.md:61]: may -> might and do not -> don't

**Review by ashwinkumar12345 — APPROVED** [review]
LGTM

File: _dashboards/reporting.md (hunk 1 above)
Comment (keithhc2) [reply _dashboards/reporting.md]: Updated to say that characters don't load because of missing fonts instead of sticking with a hard byte limit.

**Review by AMoo-Miki — APPROVED** [review]

**Pull request merged**